crates/bevy_transform/src/components/transform.rs

```diff
@@ -383,6 +383,20 @@ impl Transform {
         self.rotate(rotation);
     }
 
+    /// Flips the scale of this [`Transform`] based on which axes are not zero in the given `Dir3`.
```

Comment:
An example showcasing what will happen would be nice, e.g. flipping by `Vec3::Z` and then showing the result through an `assert_eq!`.

Reply:
I should add this to the docs real quick and then I think we should be good.